packages/colorpickers/src/styled/ColorDialog/StyledTooltipBody.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { TooltipModal } from '@zendeskgarden/react-modals';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'colorpickers.colordialog_tooltipmodal_body';
+
+export const StyledTooltipBody = styled((TooltipModal.Body as unknown) as 'div').attrs({
```

Comment:
can this be simplified to `styled(TooltipModal.Body as 'div')` by using `as: 'div'` in the attrs?